samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,182 @@
+.. _smp_svr_sample:
+
+SMP Server Sample with external XIP
+###################################
+
+Overview
+********
+
+This sample application implements a Simple Management Protocol (SMP) server whitch demonstrating spliting the appliucationso that it partially resides on internal flash and partially on QSPI flash.
+The appication uses original zephyr-rtos smp_svr sample code.
+Refer to zephyr-rtos sample documnetation for information about SMP features implemented by the sample.
```

Comment:
Add link to upstream documentation.

Reply:
@annwoj How to add link to sdk-zephyr doc?

Reply:
Please try using: `:zephyr:code-sample:`smp-svr`, in the current state I think it's just missing the first colon